net-analyzer/suricata/metadata.xml

```diff
@@ -12,6 +12,7 @@
     <flag name="control-socket">Enable unix socket</flag>
     <flag name="cuda">Enable NVIDIA Cuda computations support</flag>
     <flag name="detection">Enable detection modules</flag>
+    <flag name="hyperscan">Enable Hyperscan support</flag>
```

Comment:
I would very much prefer a more detailed description here. Yes, I know some of the existing ones are very terse too.